canonical-web-and-design/vanilla-framework
# Add dynamic blog feed to homepage

## Done

Added https://github.com/canonical-web-and-design/latest-news to pull in the most recent Vanilla articles and display them on the homepage

Drive-by: use `versioned_static` to get URLs of all static resources

Fixes #2903 
Fixes #2959

## QA

- Open [demo](https://vanilla-framework-3663.demos.haus)
- Scroll down to the "Latest news from our blog" section
- See that the content of the posts is different from those currently on [the live site](https://vanillaframework.io/), but share the same design/layout

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
